src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -63,47 +58,59 @@ def test_no_space_cont_create(self):
                 except DaosApiError as excep:
                     if not (err in repr(excep) or err2 in repr(excep)):
-                        self.d_log.error("caught exception while writing "
-                                         "object: {0}".format(repr(excep)))
+                        self.log.error("caught exception while writing "
+                                       "object: {0}".format(repr(excep)))
+                        cont.close()
                         self.fail("caught exception while writing object: {0}"
                                   .format(repr(excep)))
                     else:
-                        self.d_log.debug("pool is too full for {0} byte "
-                                         "objects".format(obj_sz))
+                        self.log.info("pool is too full for {0} byte "
+                                      "objects".format(obj_sz))
                         break
 
-        self.d_log.debug("closing container")
-        self.cont.close()
-        self.d_log.debug("closed container")
+        self.log.info("closing container")
+        cont.close()
+
+        # query the pool
+        self.log.info("Pool Query after filling")
+        self.pool.set_query_data()
+        self.log.info(
+            "Pool %s query data: %s\n", self.pool.uuid, self.pool.query_data)
+
         # create a 2nd container now that pool is full
         try:
-            self.d_log.debug("creating 2nd container")
-            self.cont2 = DaosContainer(self.context)
-            self.cont2.create(self.pool.pool.handle)
-            self.d_log.debug("created 2nd container")
+            self.log.info("creating 2nd container")
+            cont2 = DaosContainer(self.context)
+            cont2.create(self.pool.pool.handle)
+            self.log.info("created 2nd container")
 
-            self.d_log.debug("opening container 2")
-            self.cont2.open()
-            self.d_log.debug("opened container 2")
+            self.log.info("opening container 2")
+            cont2.open()
 
-            self.d_log.debug("writing one more object, write expected to fail")
-            self.cont2.write_an_obj(my_str, my_str_sz, dkey, akey,
-                                    obj_cls="OC_SX")
+            self.log.info("writing one more object, write expected to fail")
+            cont2.write_an_obj(my_str, my_str_sz, dkey, akey,
+                               obj_cls="OC_SX")
+            self.log.info("closing container")
+            cont2.close()
             self.fail("wrote one more object after pool was completely filled,"
                       " this should never print")
         except DaosApiError as excep:
             if not (err in repr(excep) or err2 in repr(excep)):
-                self.d_log.error("caught unexpected exception while "
-                                 "writing object: {0}".format(repr(excep)))
+                self.log.error("caught unexpected exception while "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions